internal/security/gnxi/rpcexec.go

```diff
@@ -0,0 +1,665 @@
+package gnxi
+
+import (
+	"context"
+	"io"
+	"strings"
+
+	"github.com/openconfig/gnoi/system"
+	"github.com/openconfig/gnsi/authz"
+	gribi "github.com/openconfig/gribi/v1/proto/service"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ygnmi/ygnmi"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	gpb "github.com/openconfig/gnmi/proto/gnmi"
+)
+
+// function AllRPC implements a sample request for service * to validate if authz works as expected.
```

Comment:
function comments should start with the name of the function, e.g. "AllRPC implements ..."